### DISCUSSION TITLE
Return the barcode polygon in QR Reader

### DISCUSSION THREAD

**Discussion by tacman** [chillerlan/php-qrcode#171] [General]
I'm working on an application where the location of the barcodes within an image is important, as the barcodes are related..  Also, when I'm done processing them image, I want to remove the barcodes (or crop the image to exclude them).

In both these cases I'd like to have the polygon (or rectangle) with the decoded barcode text.  Is this possible?  

Thanks.  Great library, btw.  Thanks for creating it.

**Comment by tacman** [comment]
On a related note, does the reader support multiple QR codes in a single image?

**Comment by codemasher** [comment]
Hi! The QR Reader is an 1:1 port of the [ZXing reader for mobile](https://play.google.com/store/apps/details?id=com.google.zxing.client.android&hl=en&gl=US&pli=1) and it has basically the same features - it reads an already cropped image of a QR Code - not more. So you have to figure out the part of the source image that includes the QR and feed it to the reader (which possibly also answers your 2nd question).
